example/android/src/main/java/com/splendo/mpp/example/LocationActivity.kt

```diff
@@ -24,13 +26,17 @@ class LocationActivity : AppCompatActivity() {
     @InternalCoroutinesApi
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+
         setContentView(R.layout.activity_main)
         if (ContextCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION)
             == PackageManager.PERMISSION_GRANTED) {
             flowLocation()
         } else {
             ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.ACCESS_FINE_LOCATION), LOCATION_PERMISSION_REQUEST_CODE);
         }
+
+        info.text = helloAndroid() + " " + helloCommon()
```

Comment:
It doesn't suggest this for function calls since they need to be wrapped in `${}`.

And maybe rightfully so, it's actually less characters this way, even with the liberal spacing I use:
```kotlin
helloAndroid() + " " + helloCommon()
"${helloAndroid()}  ${helloCommon()}"
```
I wouldn't call the second one more idiomatic for this reason.